teambit.gcp/storage/storage.ts

```diff
@@ -1,16 +1,41 @@
 import fetch from 'node-fetch';
 import { getAgent } from '@teambit/toolbox.network.agent';
+import { Client as MinioClient, BucketItem, BucketStream } from 'minio';
 
 
 export class GcpStorage {
+  private minioClient: MinioClient
+
   constructor(
     private bucketName: string,
     private proxyConfig?: {},
-    private storageAPI = 'https://storage.googleapis.com'
+    private storageAPI = 'storage.googleapis.com',
+    private accessKey?: string,
+    private secretKey?: string
   ) {}
 
+  private getMinioClient() {
+    if (this.minioClient) return this.minioClient;
+    if (!this.accessKey || !this.secretKey) {
+      throw new Error('accessKey and secretKey are required for GCP storage');
+    }
+    this.minioClient = new MinioClient({
+      endPoint: this.storageAPI,
```

Comment:
endpoint

Reply:
It's minio option name can't change it, see - https://www.npmjs.com/package/minio